File: Bike2Beans/Models/DTOs/RouteDetailsDto.cs

using Bike2Beans.Entities;
namespace Bike2Beans.Models.Dtos;


public record RouteDetailsDto(
    string Id,
    string Name,
    List<double> StartLocation,
    List<double>? EndLocation,
    List<CoffeeShopDto> RouteStops,
    double Mileage
// route?????
);

File: Bike2BeansUI/src/Data/RouteDetailsDto.ts
import { CoffeeshopDto } from './CoffeeshopDto'


export type RouteDetailsDto = {
    Id: string;
    Name: string;
    StartLocation: number;
    EndLocation: number;
    RouteStops: [CoffeeshopDto];
    Mileage: number;
}
